website/static/js/registrationUtils.js

```diff
@@ -485,11 +485,39 @@ RegistrationEditor.prototype.submit = function() {
 						url: url,
 						data: {node: currentNode, uid: currentUser.id},
 						success: function(response) {
-							bootbox.alert("Registration submitted for review!", function(result) {
-								window.location.href = '/' + currentNode.id + '/registrations/';
+							bootbox.dialog({
```

Comment:
@barbour-em I would like to see this text moved into a knockout template. You can use ko.renderTemplate, and inject the HTML into the bootbox modal.

An example of this is here: https://github.com/CenterForOpenScience/osf.io/blob/c8ea37a614b581a1e28daf859c9f21a35e84da04/website/static/js/metadata_1.js#L17